src/screens/SignUpScreen.tsx

```diff
@@ -1,78 +1,98 @@
+import { Button, Center } from 'native-base'
+import { RefObject, useRef } from 'react'
 import { TextInput } from 'react-native'
 
-import { Box, Button, Center, ControlledCheckbox, ControlledInput } from '~components'
+import { ControlledCheckbox, ControlledInput } from '~components'
 import { REGEX } from '~constants'
-import { useRef, useSignUpForm, useTheme, useTranslation } from '~hooks'
+import { useSignUpForm, useTranslation } from '~hooks'
+
+type SignUpFields = {
+  email: RefObject<TextInput>
+  password: RefObject<TextInput>
+}
 
 export const SignUpScreen = () => {
-  const { s } = useTheme()
   const { t } = useTranslation()
-  const emailInputRef = useRef<TextInput>(null)
+
+  const fields: SignUpFields = {
+    email: useRef<TextInput>(null),
+    password: useRef<TextInput>(null),
+  }
 
   const { control, errors, submit, isSubmitting } = useSignUpForm()
 
   return (
-    <Center px={8} flex={1} bg="gray100">
+    <Center px={8} flex={1}>
       <ControlledInput
-        fullWidth
-        label={t('common.user_label')}
+        mb={2}
+        autoCapitalize="none"
         control={control}
         errors={errors}
-        placeholder={t('common.user_placeholder')}
         name="user"
+        label={t('common.user_label')}
+        placeholder={t('common.user_placeholder')}
         returnKeyType="next"
-        containerStyle={[s.mB2]}
+        onSubmitEditing={fields.email.current?.focus}
       />
       <ControlledInput
-        fullWidth
+        ref={fields.email}
+        autoCapitalize="none"
         label={t('common.email_label')}
         control={control}
         errors={errors}
         keyboardType="email-address"
         placeholder={t('common.email_placeholder')}
         name="email"
-        type="email"
         returnKeyType="next"
-        onSubmitEditing={emailInputRef.current?.focus}
+        onSubmitEditing={fields.password.current?.focus}
         rules={{
           required: t('form.required'),
           pattern: {
             value: REGEX.EMAIL,
             message: t('form.invalid_email_format'),
           },
         }}
-        containerStyle={[s.mB2]}
+        isRequired
+        mb={2}
       />
       <ControlledInput
```

Comment:
I think that some of the props set on ControlledInput could be set as default inside of the component. What do you think?

Reply:
I don't see any common props to make them set with some default value - any suggestions? 🤔